src/services/audius-backend/Cognito.ts

```diff
@@ -0,0 +1,61 @@
+import AudiusBackend, {
+  AuthHeaders,
+  IDENTITY_SERVICE
+} from 'services/AudiusBackend'
+import { waitForLibsInit } from 'services/audius-backend/eagerLoadUtils'
+
+// @ts-ignore
+const libs = () => window.audiusLibs
+
+type ResponseError = {
+  statusCode?: number
+  object?: { message: string }
+}
+
+type CognitoSignatureResponse = { signature: string }
+export const getCognitoSignature = async () => {
```

Comment:
@sddioulde I'm moving this out to typescript, lmk what you think (note: I left the existing code in AudiusBackend and am not consuming this yet)

Reply:
thanks!